packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -133,6 +133,151 @@ def test_if_marathon_app_can_be_deployed_with_mesos_containerizer(dcos_api_sessi
         *test_helpers.marathon_test_app(container_type=marathon.Container.MESOS))
 
 
+def test_if_marathon_app_can_be_deployed_with_nfs_csi_volume(dcos_api_session: DcosApiSession) -> None:
+    """Marathon app deployment integration test using an NFS CSI volume.
+
+    This test verifies that a Marathon app can be deployed which attaches to
+    an NFS volume provided by the NFS CSI plugin. In order to accomplish this,
+    we must first set up an NFS share on one agent.
+    """
+
+    # We will run an NFS server on one agent and an app on another agent to
+    # verify CSI volume functionality.
+    if len(dcos_api_session.slaves) < 2:
+        pytest.skip("CSI Volume Tests require a minimum of two agents.")
+
+    expanded_config = test_helpers.get_expanded_config()
+    if expanded_config.get('security') == 'strict':
```

Comment:
Does it mean we are not checking NFS on EE at all?

Reply:
Yes.